src/App.js

```diff
@@ -1,34 +1,20 @@
-import React, { useState } from "react";
+import React from "react";
 import Dashboard from "./Dashboard/Dashboard";
-import { DEFAULT_DATA } from "./appConfig";
 import "./App.css";
 import LoginPage from "./Views/LoginPage/LoginPage/LoginPage";
-import { login } from "./api";
-import { logout } from "./api";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
 
-import { store } from "./store/store";
-import { Provider } from "react-redux";
-
-function App() {
-   const [isLoggedIn, setIsLoggedIn] = useState(false);
-
-   const loginUser = (email, password) => {
-      const loginState = login(email, password);
-      setIsLoggedIn(loginState);
-   };
+function App({ appState }) {
+   return <div className="App">{appState.isLoggedIn ? <Dashboard /> : <LoginPage />}</div>;
+}
 
-   const logoutUser = () => {
-      const logoutState = logout("");
-      setIsLoggedIn(logoutState);
-   };
+const mapStateToProps = (appState) => {
+   return appState;
+};
```

Comment:
Please, return only isLoggedIn prop.and not complete state

Reply:
const mapStateToProps = (state) => {
  const { appState } = state;
  return { ...appState, isLoggedIn: appState.isLoggedIn };
};